### ISSUE TITLE
deeplab_v3b_plus_wideresnet_citys training launch script missing

### ISSUE THREAD

**Issue by OliverColeman** [dmlc/gluon-cv#944]
The training launch script link for `deeplab_v3b_plus_wideresnet_citys` at https://gluon-cv.mxnet.io/model_zoo/segmentation.html does not work. The link is https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/logs/segmentation/deeplab_v3b_plus_wideresnet_citys.sh

**Comment by bryanyzhu** [comment]
To reproduce the model accuracy, it needs pre-training on Mapillary Vista dataset and some other techniques. So right now, this model only supports inference. But we are working on it, to make it at least compatible with the default training script.
